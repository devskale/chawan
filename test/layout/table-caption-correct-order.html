<!DOCTYPE html>
<span>test<div style="display: table-caption; background: red">caption</div><br><span>test</span></span>
