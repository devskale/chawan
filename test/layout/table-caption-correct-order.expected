test[48;2;255;0;0mcaption[49m
test
